javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable valid-typeof */
+/* exported filterOutStrings */
+function filterOutStrings(values) {
+  var arr = [];
+  var number = typeof 8;
+  for (var i = 0; i < values.length; i++) {
+    if (typeof values[i] === number) {
```

Comment:
This will only work if we are only dealing with numbers. The goal of this function is to filter out strings from all data types, so you really want to search for anything that is not a string 👍

Reply:
yeah after the discussion i see why and what i could have done differently .. thanks for pointing that out